player/lua/osc.lua

```diff
@@ -2053,10 +2054,13 @@ function osc_init()
             "absolute-percent", "exact") end
     ne.eventresponder["reset"] =
         function (element) element.state.lastseek = nil end
-    ne.eventresponder["wheel_up_press"] =
-        function () mp.commandv("osd-auto", "seek",  10) end
-    ne.eventresponder["wheel_down_press"] =
-        function () mp.commandv("osd-auto", "seek", -10) end
+
+    if user_opts.mousewheelseek then
+        ne.eventresponder["wheel_up_press"] =
+            function () mp.commandv("osd-auto", "seek",  10) end
+        ne.eventresponder["wheel_down_press"] =
+            function () mp.commandv("osd-auto", "seek", -10) end
+    end
```

Comment:
Conversely you could argue that seeking on the volume knob interferes with having wheel seek inside the video. But nobody ever complained about that :sunglasses: 

what I'm saying: this option is too specific and *if we need to* I'd rather have one that disables all wheel bindings